floodviz/test/hydrograph_utils_tests.py

```diff
@@ -0,0 +1,84 @@
+import unittest
```

Comment:
Should have mentioned this earlier, please put this file in floodviz/tests/test_hydrograph_utils.py. This is our practice here.